tests/all/integration/api/helpers/test_systemnotifications.py

```diff
@@ -0,0 +1,258 @@
+import unittest
+
+from app import current_app as app
+from tests.all.integration.utils import OpenEventTestCase
+from app.api.helpers.system_notifications import (
+    get_event_exported_actions,
+    get_event_imported_actions,
+    get_monthly_payment_notification_actions,
+    get_monthly_payment_follow_up_notification_actions,
+    get_ticket_purchased_notification_actions,
+    get_ticket_purchased_attendee_notification_actions,
+    get_ticket_purchased_organizer_notification_actions,
+    get_event_published_notification_actions,
+    get_event_role_notification_actions,
+    get_new_session_notification_actions,
+    get_session_schedule_notification_actions,
+    get_next_event_notification_actions,
+    get_session_accept_reject_notification_actions,
+    get_invite_papers_notification_actions
+)
+from tests.all.integration.setup_database import Setup
+from app.models.notification import NotificationAction
+
+
+class TestSystemNotificationHelperValidation(OpenEventTestCase):
+    def setUp(self):
+        self.app = Setup.create_app()
+
+    def test_event_exported(self):
+        """Method to test the actions associated with a notification about an event being successfully exported."""
+
+        with app.test_request_context():
+            request_url = 'https://localhost/some/path/image.png'
+            response = get_event_exported_actions(request_url)
+            expected_action = NotificationAction(
+                subject='event-export',
+                link=request_url,
+                action_type='download'
+            )
+            self.assertIsInstance(response, list)
+            self.assertEqual(expected_action.subject, response[0].subject)
```

Comment:
Test counts as well. Better test whole list

https://stackoverflow.com/questions/12813633/how-to-assert-two-list-contain-the-same-elements-in-python

Reply:
I think we can't use `assertCountEqual` on a list of objects. `assertCountEqual` uses collections `Counter` internally to count the elements for hashable object types and creates a dictionary with with each key as element and its count as value. Since in our case both the object as in string aren't equal so the keys differ while the count of the key is same and the test fails on using `assertCountEqual`.

Reply:
OK, then just assert counts and content manually

Reply:
I have calculated the list length using `len` function and asserted using `assertTrue`, in case when the length is different, it will return `False` and fail the test.

Reply:
Use assertEquals

Reply:
My bad, I was implementing it in another way in the start and didn't thought of `assertEqual`, I'll keep note of these small things in further PR's.